_inc/client/components/non-admin-view/index.jsx

```diff
@@ -18,7 +18,9 @@ const NonAdminView = React.createClass( {
 	renderContent: function() {
 		const userData = window.Initial_State.userData.currentUser;
 		const isLinked = this.props.isLinked( this.props );
-		let headerText = 'Write posts via email, get notifications about your site activity, and log in with a single click.';
+		let headerText = userData.permissions.edit_posts
+			? 'Write posts via email, get notifications about your site activity, and log in with a single click.'
```

Comment:
We chatted in Slack and Derek gave us some helpful links. I'd like to address in a separate PR though.